css/properties.json

```diff
@@ -3940,15 +3940,15 @@
     "syntax": "<color>",
     "media": "visual",
     "inherited": true,
-    "animationType": "color",
+    "animationType": "byComputedValueType",
     "percentages": "no",
     "groups": [
       "CSS Color"
     ],
-    "initial": "variesFromBrowserToBrowser",
-    "appliesto": "allElements",
-    "computed": "translucentValuesRGBAOtherwiseRGB",
-    "order": "uniqueOrder",
+    "initial": "canvastext",
```

Comment:
I opted for the lowercased "canvastext" to be in line with for example "currentcolor" in the file